app/grandchallenge/evaluation/models.py

```diff
@@ -475,6 +483,25 @@ class Phase(FieldChangeMixin, HangingProtocolMixin, UUIDModel):
         blank=True,
         help_text="The output interfaces that the algorithms for this phase must use",
     )
+    selectable_gpu_type_choices_evaluation: list[str] = models.JSONField(
```

Comment:
Please do not add type annotations, they are not checked in this repo and the recommendation from mypy for integration is to always start without type hints:

```suggestion
    selectable_gpu_type_choices_evaluation = models.JSONField(
```